apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/env/dirprovider/ConfigDirProviderFactory.java

```diff
@@ -1,29 +1,77 @@
 /*
  * Copyright © 2018 Apollo Foundation
  */
-
 package com.apollocurrency.aplwallet.apl.util.env.dirprovider;
 
+import com.apollocurrency.aplwallet.apl.util.FileUtils;
 import com.apollocurrency.aplwallet.apl.util.env.RuntimeEnvironment;
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
 
 /**
  * Simple factory which should used for configDirProvider creation
  */
 public class ConfigDirProviderFactory {
+
     private static boolean isService;
     private static String applicationName;
     private static int netIdx;
+    private static String uuid_or_part;
+    private static UUID uuid;
 
-    public static void setup(boolean isServiceP, String applicationNameP, int netIdxP) {
+    public static void setup(boolean isServiceP, String applicationNameP, int netIdxP, String uuid_or_partP) {
```

Comment:
What about the camelCase naming convention?

Reply:
fixed